printer_mods/SinisterRj/SafetyPrinter/readme.md

```diff
@@ -0,0 +1,16 @@
+# SafetyPrinter Mod
+[Safety Printer](https://github.com/SinisterRj/SafetyPrinter) is a great project to improve the safety of your printer. The project aims to improve the safety of 3D FDM Printers by adding an external MCU (an Arduino) which increases your chance of preventing or mitigating a fire at your home. 
```

Comment:
Everything is explained in this repo's wiki, So I decided to keep things concise here to avoid any miss information if, in a near future, things change on the project. This mod is just an addendum to the main project. So, if you don't mind I wish to keep this description brief, but if you still think that is necessary to add something i do it.

Reply:
I don't think you need to duplicate the entire wiki in her...  but a little more information would be useful for people looking through the mods, this description doesn't really tell me anything about it.

Reply:
ok, I'll improve it.